city_scrapers/spiders/cinoh_city_council.py

```
@@ -0,0 +1,94 @@
+
+    def _parse_links(self, obj):
+
+        links = []
+
+        if obj.get("Name"):
+            links.append({"title": "meeting page", "href": obj["Name"]["url"]})
+
+        if obj.get("iCalendar"):
+            links.append({"title": "iCalendar", "href": obj["iCalendar"]["url"]})
+
+        if not obj.get("Meeting Details") == "Meeting\u00a0details":
```

Ruff (0.8.2)

Use `obj.get("Meeting Details") != "Meeting\xa0details"` instead of `not obj.get("Meeting Details") == "Meeting\xa0details"`

Replace with `!=` operator

(SIM201)